### PR TITLE
[bitnami/kibana] Add Kibana-server SSL and Elasticsearch SSL

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

Adds support for Elasticsearch X-Pack security features (Basic authentication and encrypted REST API) #6486.

Additionally, the Kibana Web server can now be configured with TLS certificates to serve clients via HTTPS (`server.ssl` settings). It is not related to X-Pack Security and does not need Elasticsearch to be configured with SSL

**Benefits**

Additional options to communicate with Elasticsearch.
- Adds Elasticsearch SSL settings:
    - Trusted CAs in PEM or PKCS12 format.
    - Verification mode.
- Adds Kibana credentials to log into Elasticsearch (kibanaUser:kibanaPassword). **Note:** this is the user used by Kibana to manage things such as the `.kibana` index, where it stores visualizations, dashboards, and settings. It is not necessarily the same user used to log into the UI.

Kibana Web server SSL settings.
- The kibana server can be configured using PKCS12 or PEM certificates.
- Adds support for autoGenerated certificates (not recommended for production environments).
 
**Checklist** 

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by migruiz4** [comment]
Pending:
- Add a section documenting the new features and values in the README.md
- Add validations.

File: bitnami/kibana/templates/deployment.yaml [hunk 1]
```diff
@@ -13,9 +13,13 @@ spec:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- if .Values.podAnnotations }}
-      annotations: {{- include "common.tplvalues.render" (dict "value" .Values.podAnnotations "context" $) | nindent 8 }}
-      {{- end }}
+      annotations:
```
Comment (alvneiayu) [inline bitnami/kibana/templates/deployment.yaml:16]: If you don't set up podAnnotations and you don't use TLS, I think that annotations will be empty no? Is it works?

**Review by alvneiayu — CHANGES_REQUESTED** [review]

**Review by alvneiayu — APPROVED** [review]
LGTM

**Review by alvneiayu — APPROVED** [review]
LGTM

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**